app/schemas/models/standards_correlation.schema.js

```diff
@@ -0,0 +1,90 @@
+const c = require('./../schemas')
+
+const StandardsCorrelation = c.object(
+  {
+    description: '',
+    title: 'Standards Correlation',
+    required: ['name']
+  },
+  {
+    subject: c.shortString({
+      title: 'Subject',
+      type: 'string',
+      enum: ['computer-science', 'literacy', 'math', 'science']
+    }),
+    district: c.objectId({
+      links: [{ rel: 'extra', href: '/db/district/{($)}' }],
+      title: 'District ID',
+      description: 'The ID of the district that this correlation belongs to'
+    }),
+    administrativeRegion: c.shortString({
+      title: 'Administrative Region',
+    }),
+    country: c.shortString({
+      title: 'Country',
```

Comment:
Are we going to have this be a drop-down from our countries list, so that we can get the country formatting correct every time?